script/Ciel/Day 14/14_01B_03_CIEL14B_6.txt

```diff
@@ -567,92 +619,99 @@ I don't know what you have in mind―――"
+I can't sense any kind of will emanating from Arcueid anymore.
 }
 [sha:d3f83e45e9c4110aaeb390c02af0c64f14e9bbd2]{
 -- Page 327, Offset 41198.
 -- 先輩への殺意も、俺への怒りも、ロアへの狂気も。
-Whether it's her intent to kill Senpai, her anger against me, or her obsession with Roa.
+No desire to kill Senpai,
+ no anger directed at me,
+ no hint of her obsession with Roa.
 }
 [sha:627e514e77080e2733aebc978bb253cea0396e2a]{
 -- Page 327, Offset 41199.
 -- そして、彼女自身の、存在への強い意志も。
-Even her strong will to exist is nowhere to be seen.
+Not even her fierce will to continue to exist.
 }
 [sha:c01f58901e373ca6389bace1e7a3c5124fc3505e]{
 -- Page 328, Offset 41200.
 -- 「わたし、そろそろ帰るわ。
-"I'm going to go home now.
+"I'm about to head home.
 }
 [sha:82001a1c815a4c0223ed0045724be9c1ccb9cca6]{
 -- Page 328, Offset 41201.
 -- いいかげん眠くなってきちゃったし」
-Everything has made me awfully sleepy."
+This has all made me awfully sleepy."
 }
 [sha:c5024f5bedd7b6f48123c675df8d44cde4217a4e]{
 -- Page 328, Offset 41202.
 -- 「え……え？」
-"Eh... eh?!"
+"Huh... %{i}Huuuh%{/i}?!"
 }
 [sha:7d0001597a4d15f02c73dc2932e87c94be921971]{
 -- Page 329, Offset 41203.
 -- 困惑するシエル先輩。
-Ciel-senpai is bewildered.
+Ciel-senpai is completely bewildered.
 }
 [sha:7716f96307b3a4c9b501ce7708ea60e986b29b81]{
 -- Page 329, Offset 41204.
 -- 目が覚めればロアは消え、最大の敵も去ろうとしている。
-As soon as she wakes up, Roa had vanished, and her greatest enemy is also going away.
+She woke up only to find that Roa was already gone,
+ and her greatest enemy is also about to head out.
 }
 [sha:c332c7cd88d6ce5c09778992de8d4d57d18e4d38]{
 -- Page 329, Offset 41205.
 -- 本来なら喜ぶべき事だ。けど、それを口にするアイツがあまりにも晴れやかで、目を疑っている。
-Normally she would be happy about all of this. But the way Arcueid said it was so bright that it made her doubt her own eyes.
+Normally she'd be thrilled, but Arcueid seems so carefree that she can hardly believe it's real.
 }
 [sha:8a20f3e09cbc01b7d25cc1dad3033bd137d1a099]{
 -- Page 330, Offset 41206.
 -- 「で、でも―――！　貴女も、彼を」
-"B-But―――! What about him?"
+"B-But!
+ Don't you... What about him?"
 }
 [sha:7c7fb73066e57a661aab2975a19d6d0338b30f71]{
 -- Page 330, Offset 41207.
 -- 「そんな意地悪なひと、どうでもいい」
-"I don't care about that awful person anymore."
+"I don't care about that bully anymore."
 }
 [sha:c4a89d25215b4a1d6aa667feca203ecad3be7710]{
 -- Page 330, Offset 41208.
 -- 金の髪がなびく。
-Her golden hair sways.
+Her golden hair flutters.
 }
 [sha:93363626e04ee1c8e892bbf3cef43bc1b5c55cf2]{
 -- Page 330, Offset 41209.
 -- 白い吸血姫は、自分のいるべき場所に踵を返し、
-The white vampiress turns to the place she must return.
+The vampiress in white turns to face toward the place she must return to.
 }
 [sha:99dc9ebb2b92d9176dd94d21c667ad3fe46c6c87]{
 -- Page 331, Offset 41210.
 -- 「シエルにあげるわ。―――大切にしてね」
-"I'll let you have him. ―――You better treat him well."
+"I'll let you have him.
+ You better take care of him, okay?"
 }
 [sha:0bb69b6bac839c316eeb70ea2d3eceee0b5a776d]{
 -- Page 331, Offset 41211.
 -- ……俺と同じ。
-...The same could also be said for me about her.
+The same goes for me...
```

Comment:
Huh... I can't figure if this is JUST meant to connect with 41209 or be the lead-in to 41212 as well.
>———Just as I must return to where I belong too.

>Just like me———

>She smiles, wishing for the happiness of another, and vanishes into the night.